roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -121,6 +121,13 @@ server {
     deny all;
   }
   {% endblock %}
+  
+  {% block blade_twig_templates -%}
+  # Prevent Blade and Twig templates from being accessed directly.
+  location ~* \.(blade\.php|twig)(/.*)?$ {
```

Comment:
Why the `(/.*)`? This is going to match a lot more than just files ending in `.blade.php` or `.twig`

Reply:
see https://github.com/roots/docs/pull/180#issuecomment-474637910

Reply:
🤔 okay but I still don't understand that situation. Maybe @QWp6t can clear it up

Reply:
Meh, you're probably right in terms of it not being an issue in Trellis unless the end-user messed with nginx configuration. Do you just want to leave it as a safe-guard or revert it?

Reply:
The twig case is the one that worries me slightly more since there's no `.php` in it as well. So `example.com/foo.twig/asdf` would match? It likely doesn't matter, but I don't want to break any URLs which might be valid.

Reply:
It would indeed match– but as far as WordPress permalinks go it'd be a non-issue as `.` is replaced with `-` when editing a slug.

Reply:
@swalkinshaw I don't think `(/.*)` is needed here.

https://roots-example-project.com/app/themes/sage/resources/views/index.blade.php/more-stuff-here

Seems to work fine.

I was only suggesting it for Apache because after testing on a few Apache servers, they all displayed the blade file.